Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gem 'rails', '5.1.4'
 
 gem 'bootstrap-kaminari-views', '~> 0.0.5'
+gem 'foreman', '~> 0.84'
```

Comment:
What's this being used for? I know nothing about foreman, other than what I've just read on the project page [1], and one of the first things that says is "Ruby users should take care not to install foreman in their project's Gemfile.". I've got nothing against doing something different, but it would be good to know why?

1: https://github.com/ddollar/foreman